metadata-models/src/main/pegasus/com/linkedin/chart/ChartType.pdl

```diff
@@ -0,0 +1,32 @@
+namespace com.linkedin.chart
+
+/**
+ * The various types of charts
+ */
+enum ChartType {
```

Comment:
are these types referenced from Looker or internally? BAR chart can be of multiple formats so we might want to check Looker to see the types supported there?

Reply:
Referred to Looker charts: https://docs.looker.com/exploring-data/visualizing-query-results/visualization-types

Reply:
Looks like the possibility is huge https://www.edrawsoft.com/chart-types-uses.html (we can expand it later).
Are we only limiting to looker / redash charts?